### ISSUE TITLE
beit on multi-label classification

### ISSUE THREAD

**Issue by JiweiZh** [microsoft/unilm#439]
I am using beit pretrained and fine-tuning on imagenet 21k, I see the output classifier layer is 21841, do I need to add more layers for multi-label classification?  and what's the loss you use on pretrained and fine-tuning, thx.

**Comment by addf400** [comment]
We use cross-entropy loss for pretraining and fine-tuning. In addition, we set label smoothing 0.1 for fine-tuning. 
During the fine-tuning, we employ a simple linear classifier as the task layer. Specifically, we use average pooling to aggregate the representations and feed the global to a softmax classifier. 
You can try to use sigmoid loss to replace cross-entropy loss for multi-label classification.
